cgroups/fs/devices.go

```diff
@@ -10,7 +10,7 @@ type DevicesGroup struct {
 
 func (s *DevicesGroup) Apply(d *data) error {
 	dir, err := d.join("devices")
-	if err != nil {
+	if err != nil && !cgroups.IsNotFound(err) {
```

Comment:
We really cannot do this because devices is a hard requirement.  Without the devices cgroup it is a security issue because a container will have unrestricted access to device nodes.